File: bancovsgao.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int q = input.nextInt();
        long tongsogao = 0;
        long gao = 1;

        for (int i = 0; i < n; i++) {
            tongsogao += gao;
            gao *= q;
        }
System.out.println(tongsogao);
}
}

File: bancovsgao.cpp
#include <iostream>

using namespace std;

int main() {
    int n, q;
    cin>>n>>q;
    long long tongsogao = 0;
    long long gao = 1;

    for (int i = 0; i < n; i++) {
        tongsogao += gao;
        gao *= q;
    }

    cout  << tongsogao;

    return 0;
}
